hipify_cli.py

```diff
@@ -50,12 +50,6 @@ def main():
         help="Source files to be excluded for hipify",
         required=False)
 
-    parser.add_argument(
```

Comment:
This PR removes some internal changes related to CuPy. Please don't remove it. This will impact other frameworks.

Reply:
Thanks for reviewing, @lcskrishna. I agree we should keep these changes. @liligwu Any reason to change hipify_cli.py? It doesn't even exist in PyTorch upstream, so it isn't about conflicts...

Reply:
@liligwu Interesting, just saw the codebase of fbgemm looks like its using CMake API. There were some changes after we added custom map in CMake way. I would recommend rebase your code with latest hipify_torch rather than removing it altogether. This might impact xformers as well. 
cc: @jithunnair-amd

Reply:
I am working on modifying the changes in this PR to reinstate some of the features we had added to hipify_torch. The `custom_map_list` was one of those features and it involved an addition of a keyword argument to `hipify`. It'll be part of the changes I'll reinstate